x-pack/plugins/console_extensions/spec/generated/ccr.follow.json

```diff
@@ -0,0 +1,11 @@
+{
+  "ccr.follow": {
+    "methods": [
+      "PUT"
+    ],
+    "patterns": [
+      "{indices}/_ccr/follow"
```

Comment:
Drive-by comment, we only handle a single index here. Not sure if indices is an indication we handle multiple, so want to clarify in case.

Reply:
That was generated from REST specs, but console treats {indices} the same as {index} and does not have true support for multiple index name completion.

Reply:
Great. Thanks for clarifying.